src/yanfeishao/cs555/data/data.ged

```diff
@@ -224,7 +224,7 @@
 1 NAME John /Cook/ Jr
 1 SEX M
 1 BIRT 
-2 DATE 8  Feb 1952
+2 DATE 8 Feb 1952

```

Comment:
because there is an extra blank space there, to make my code work I need to remove the blank space in the date